HW1/P3/P3.py

```diff
@@ -0,0 +1,10 @@
+from pyspark import SparkContext
+sc = SparkContext("local", "Simple App")
+words = sc.textFile("wlist.txt")
+
+rdd = words.map(lambda x : (''.join(sorted(x)), x)) \
+		   .groupByKey() \
+		   .map(lambda (key,values) : (key, len(list(values)), list(values))) \
+		   .sortBy(lambda (x1, x2, x3) : x2, ascending = False)
+
```

Comment:
Clear style with one function call each line, and also condensed coding. The script is much more shorter than mine.